CanHazFunny/CanHazFunny/JokeService.cs

```diff
@@ -1,15 +1,21 @@
 ﻿using System.Net.Http;
+using System.Text.Json;
 
-namespace CanHazFunny
+namespace CanHazFunny;
+
+#pragma warning disable IDE1006 // Suppressing naming rule violation because the parameter name must match the json key
+public readonly record struct JokeJson(string joke);
+#pragma warning restore IDE1006
```

Comment:
```suggestion
public readonly record struct JokeJson()
{
    [JsonPropertyName("joke")]
    public string Joke { get; init; }
}
```
You can use the `JsonPropertyName` attribute in order to not break naming conventions.